lib/fluent/plugin/out_stdout.rb

```diff
@@ -25,6 +25,9 @@ class StdoutOutput < Output
     DEFAULT_LINE_FORMAT_TYPE = 'stdout'
     DEFAULT_FORMAT_TYPE = 'json'
 
+    desc "If Fluentd logger outputs logs to a file, this plugin outputs events to the file as well."
+    config_param :sync_logger, :bool, default: true
```

Comment:
We discussed that it would be better to change this parameter to `use_logger`.

Reply:
Another idea: `follow_logger_output: true`, but `use_logger` might be straightforward enough.

Reply:
Thanks!
I have changed the option name to `use_logger`!